sklearn/ensemble/_gb.py

```diff
@@ -1725,6 +1725,11 @@ def staged_predict_proba(self, X):
                 "loss=%r does not support predict_proba" % self.loss
             ) from e
 
+    def __sklearn_tags__(self):
```

Comment:
Isn't it possible to have a single `__sklearn_tags__(self)` defined in `BaseGradientBoosting`?

Reply:
done in 705c414